src/thread_task.rs

```diff
@@ -791,14 +792,18 @@ unsafe fn delayed_fg_task<'l, 'ttd: 'l>(
 pub unsafe extern "C" fn rav1d_worker_task(data: *mut c_void) -> *mut c_void {
     let tc: &mut Rav1dTaskContext = &mut *(data as *mut Rav1dTaskContext);
     let c: &Rav1dContext = &*tc.c;
-    let ttd: &mut TaskThreadData = &mut *tc.task_thread.ttd;
+
+    // We clone the Arc here to avoid an immutable borrow of tc across the call
```

Comment:
This clone is immediately dropped after the function ends, right?  Can you add that to the comment, that this Arc clone is just temporary to get around borrowck?

Reply:
Sure, done. This function has the same lifetime as the thread, but that's fine.